pkg/storage/kubernetes.go

```diff
@@ -58,7 +52,7 @@ func NewKubernetesIPAM(containerID string, ipamConf whereaboutstypes.IPAMConfig)
 	if err != nil {
 		return nil, err
 	}
-	return &KubernetesIPAM{c, ipamConf, containerID, namespace, DatastoreRetries}, nil
+	return &KubernetesIPAM{c, ipamConf, containerID, namespace, ipamConf.Name, DatastoreRetries}, nil
```

Comment:
Interesting, so instead of using the ip range as the resource name, this is utilizing the `"name"` of the CNI config (e.g. `mynet` from the tests). 

I'm not against this, we're rethinking the pool resource type in #51; this is a step in that direction I believe.